webapp/views.py

```diff
@@ -87,31 +85,31 @@ def delete(self, request, customer_id):
 
             return HttpResponse(status=200)
         except Account.DoesNotExist:
-            raise PermissionDenied("Customer does not exist")
+            raise Http404("Customer does not exist")
         
 
 class AccountOnly(APIView):
     def get(self, request, customer_id):
         try:
             account = Account.objects.filter(customer_id=customer_id)
             if not account.exists():
-                raise PermissionDenied("Account does not exist")
+                raise Http404("Account does not exist")
             
             serializer = AccountSerializer(instance=account, many=True)
             return Response(serializer.data)
         except Account.DoesNotExist:
-            raise PermissionDenied("Account does not exist")
+            raise Http404("Account does not exist")
         
     def delete(self, request, customer_id, account_number):
         try:
             account = Account.objects.filter(customer_id=customer_id, account_number=account_number)
             if not account.exists():
-                raise PermissionDenied("Account does not exist")
+                raise Http404("Account does not exist")
 
             account.delete()
             return HttpResponse(status=200)
         except Account.DoesNotExist:
-            raise PermissionDenied("Account does not exist")
+            raise Http404("Account does not exist")
```

Comment:
In general, 404 error is "page not found" error - it should not have customizable messages, simply because it should be raised only when a page is not found.